A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2461,30 +2461,22 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     async trySetAverageWind(s) {
-        const validDelims = ["HD", "H", "-", "TL", "T", "+", ""]; // Based on arrays being iterated, it will check values like "23" last
-        const matchedIndex = validDelims.findIndex(element => s.includes(element));
-
-        if (matchedIndex >= 0) {
-            const wind = parseFloat(s.split(validDelims[matchedIndex])[1]);
-
-            this._windDir = matchedIndex <= 2 ? this._windDirections.HEADWIND : this._windDirections.TAILWIND;
-
-            if (isFinite(wind)) {
-                if (this.isAvgWindInRange(wind)) {
-                    this.averageWind = wind;
-                    return true;
-                } else {
-                    this.addNewMessage(NXSystemMessages.entryOutOfRange);
-                    return false;
-                }
-            } else {
-                this.addNewMessage(NXSystemMessages.formatError);
-                return false;
-            }
-        } else {
+        const validDelims = ["TL", "T", "+", "HD", "H", "-"];
+        const matchedIndex = validDelims.findIndex(element => s.startsWith(element));
+        const digits = matchedIndex >= 0 ? s.replace(validDelims[matchedIndex], "") : s;
+        const isNum = /^\d+$/.test(digits);
+        if (!isNum) {
             this.addNewMessage(NXSystemMessages.formatError);
             return false;
         }
+        const wind = parseInt(digits);
+        this._windDir = matchedIndex <= 2 ? this._windDirections.TAILWIND : this._windDirections.HEADWIND;
+        if (!this.isAvgWindInRange(wind)) {
```

Comment:
This check could be inlined for now as it's not used in any other place.

Reply:
Removed the function. And also `wind < 0` check is not needed here. Thanks.